src/app/components/home/profile/edit-profile/edit-profile.component.ts

```diff
@@ -1,8 +1,97 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, Input, OnInit } from '@angular/core';
+import * as hash from 'object-hash';
+import { BasicUser, UserService } from 'src/app/services/user.service';
+import { AppToastService } from 'src/app/services/toast.service';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { EqualityValidator } from '../../login/EqualityValidator';
+
+export class EditUserData {
+  id: any;
+  login_name?: string;
+  display_name: string;
+  currentPassword: string;
+  newPassword?: string;
+  newPasswordConfirmed?: string;
+}
 
 @Component({
   selector: 'app-edit-profile',
   templateUrl: './edit-profile.component.html',
-  styleUrls: ['./edit-profile.component.css'],
+  styleUrls: ['../../../../../../src/assets/css/styles.css', './edit-profile.component.css'],
```

Comment:
styles.css is already included as a default. '../../../../../../src/assets/css/styles.css' should not be nessecary.